packages/api/src/routeHandlers.ts

```diff
@@ -12,33 +9,38 @@ export const getTransports: Handler = (c, req: Request, res: Response) => {
         volume: 0,
         weight: 0,
       },
-      earliestStart: '',
-      latestEnd: '',
+      earliest_start: '',
+      latest_end: '',
       metadata: {},
-      startAddress: {
+      start_address: {
         city: '',
         street: '',
         name: '',
-        position: '',
+        position: {
+          lat: 0,
+          lon: 0
+        }
       },
-      endAddress: {
+      end_address: {
         city: '',
         street: '',
         name: '',
-        lon: 0,
-        lat: 0,
+        position: {
+          lon: 0,
+          lat: 0,
+        }
       },
     },
   ])
 }
 
-export const getItinerary: Handler = (
-  _,
-  req: Request<{}, any, any, operations['getItinerary']['parameters']['query']>,
-  res: Response
-) =>
+export const get_itinerary: Handler = (
```

Comment:
It looks to me like we want to make use of the Context object rather than Request/Response. The type signatures of these three parameters are implicit since we are claiming that these are of type `Handler`